typespec-tests/src/main/java/com/cadl/enumservice/models/Operation.java

```diff
@@ -47,6 +47,27 @@ public final class Operation {
     @JsonProperty(value = "color")
     private ColorModel color;
 
+    /*
+     * The priorityValue property.
+     */
+    @Generated
+    @JsonProperty(value = "priorityValue")
+    private Priority priorityValue = Priority.LOW;
+
+    /*
+     * The colorValue property.
+     */
+    @Generated
+    @JsonProperty(value = "colorValue")
+    private Color colorValue = Color.GREEN;
+
+    /*
+     * The colorModelValue property.
+     */
+    @Generated
+    @JsonProperty(value = "colorModelValue")
+    private ColorModel colorModelValue = ColorModel.BLUE;
```

Comment:
Generated code be this. Required type is treated as constant (which means it is read-only -- no setter).